src/config/configuration.validator.spec.ts

```diff
@@ -10,13 +10,25 @@ describe('Configuration validator', () => {
     expect(validated).toBe(expected);
   });
 
-  it('should detect a malformed configuration in production environment', () => {
+  it('should detect missing mandatory configuration in production environment', () => {
     process.env.NODE_ENV = 'production';
     expect(() => validate(JSON.parse(fakeJson()))).toThrow(
-      /Mandatory configuration is missing: .*AUTH_TOKEN.*EXCHANGE_API_KEY/,
+      /must have required property 'AUTH_TOKEN'.*must have required property 'EXCHANGE_API_KEY'/,
     );
   });
 
+  it('should an invalid LOG_LEVEL configuration in production environment', () => {
+    process.env.NODE_ENV = 'production';
```

Comment:
Why do we need to set the environment to `production`? The validation should be the same in a testing and production scenarios right?

Reply:
Yes, true. It's not necessary for this test, thanks. Changed in [cde8296](https://github.com/safe-global/safe-client-gateway-nest/pull/516/commits/cde8296f1a982e7facc15879d09d5021dc8099df)